### PR TITLE
Mark endpoints for extracting to different port

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

File: src/main/scala/org/constellation/API.scala [hunk 1]
```diff
@@ -215,6 +219,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
             )
 
           } ~
+          // TODO: Different port (It is also duplicated in PeerAPI!)
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: It should stay in the PeerAPI, I think only

File: src/main/scala/org/constellation/API.scala [hunk 2]
```diff
@@ -223,11 +228,13 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
               }
             )(complete(_))
           } ~
+          // TODO: Different port
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: It can be removed I think

File: src/main/scala/org/constellation/API.scala [hunk 3]
```diff
@@ -223,11 +228,13 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
...
           path("storedReputation") {
             APIDirective.handle(
               dao.trustManager.getStoredReputation.map(TrustData)
             )(complete(_))
           } ~
+          // TODO: Different port
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: It can be removed I think

File: src/main/scala/org/constellation/API.scala [hunk 4]
```diff
@@ -238,6 +245,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
 
   val postEndpoints =
     post {
+      // TODO: Used only in tests and Cluster.addToPeer which... has no usages
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: It can be removed I think

File: src/main/scala/org/constellation/API.scala [hunk 5]
```diff
@@ -252,6 +260,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
           }
         }
       } ~
+        // TODO: Different port
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: Is it used?

Comment (kpudlik) [reply src/main/scala/org/constellation/API.scala]: Nope :)

File: src/main/scala/org/constellation/API.scala [hunk 6]
```diff
@@ -252,6 +260,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
           }
         }
       } ~
+        // TODO: Different port, used in Simulation
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: I think it's unused

File: src/main/scala/org/constellation/API.scala [hunk 7]
```diff
@@ -275,6 +284,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
               complete(StatusCodes.InternalServerError)
           }
         } ~
+        // TODO: Different port, used only in Simulation
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: I think it should stay public

File: src/main/scala/org/constellation/API.scala [hunk 8]
```diff
@@ -302,6 +312,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
             APIDirective.handle(io)(complete(_))
           }
         } ~
+        // TODO: Different port, used only in Simulation
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/API.scala]: Should be removed?

**Pull request merged**